vsts/vsts.yaml

```diff
@@ -242,6 +242,117 @@ phases:
 
     condition: always()
 
+### Windows IoT Core build ###
+
+- phase: IoTUAP
+  displayName: Windows IoT Core
+# Uncomment to serialize builds:
+#  dependsOn: WINDOWS
+
+  condition: succeeded()
+
+  queue:
+    name: win-iot-core
+    demands:
+      - e2e-tests-os -equals WinIoTCore_x64
+
+  steps:
+  - powershell: ./vsts/manual_checkout.ps1
+    displayName: 'GIT checkout $(COMMIT_FROM)'
+    env:
+      COMMIT_FROM: $(COMMIT_FROM)
+
+  - script: |
+       rem Start
+       
+       rem Start Squid
+       C:\Data\Squid\bin\squid.exe
+       
+       rem Start ETL logging
+       for /F "eol=; tokens=2 delims={}" %%i in (tools\CaptureLogs\iot_providers.txt) do @echo %%i;0xffffffffffffffff;0xff >> tools\CaptureLogs\iot_providers_temp.txt
```

Comment:
Could you please explain this section.

Reply:
The logging tool _tracelog_ that is present on Windows IoT Core accepts a different format of the file with provider GUIDs, than the _logman_ tool.
The Batch script here converts the _iot_providers.txt_ file to a format that _tracelog_ understands by processing the input text file line by line. The temporary output file (used for _tracelog_ gets deleted as soon as not needed anymore).
Input format: {\<provider guid\>}
Output format: \<provider guid\>;0xffffffffffffffff;0xff